plot-api/src/commonMain/kotlin/jetbrains/letsPlot/stat/stat_qq2.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022. JetBrains s.r.o.
+ * Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+ */
+
+package jetbrains.letsPlot.stat
+
+import jetbrains.letsPlot.Pos
+import jetbrains.letsPlot.Stat
+import jetbrains.letsPlot.intern.GeomKind
+import jetbrains.letsPlot.intern.Options
+import jetbrains.letsPlot.intern.layer.geom.PointAesthetics
+import jetbrains.letsPlot.intern.layer.geom.PointMapping
+import jetbrains.letsPlot.intern.layer.stat.QQ2StatAesthetics
+
+@Suppress("ClassName")
+class statQQ2(
+    data: Map<*, *>? = null,
+    geom: jetbrains.letsPlot.intern.layer.GeomOptions = jetbrains.letsPlot.intern.layer.GeomOptions(GeomKind.Q_Q_2),
+    position: jetbrains.letsPlot.intern.layer.PosOptions = Pos.identity,
+    showLegend: Boolean = true,
+    sampling: jetbrains.letsPlot.intern.layer.SamplingOptions? = null,
+    override val x: Double? = null,
+    override val y: Double? = null,
+    override val alpha: Number? = null,
+    override val color: Any? = null,
+    override val fill: Any? = null,
+    override val shape: Any? = null,
+    override val size: Number? = null,
+    override val stroke: Number? = null,
+    mapping: PointMapping.() -> Unit = {}
+) : PointAesthetics,
```

Comment:
Let's use QQ2Aesthetics, QQ2StatAesthetics instead of PointAesthetics.

Reply:
Correction: if the parameter `geom` in this class takes value `Geom.poinr()` as a default value, then using `PointAesthetics`, `PointMapping` in OK .

Reply:
The same applies to similar comments below.

Reply:
Done: default values for `geom` parameters changed to `Geom.point(), Geom.path()` in `statQQ, statQQ2Line`